packages/mdc-top-app-bar/constants.js

```diff
@@ -17,9 +17,17 @@
 /** @enum {string} */
 const strings = {
   NAVIGATION_EVENT: 'MDCTopAppBar:nav',
+  ROOT_SELECTOR: '.mdc-top-app-bar',
   TITLE_SELECTOR: '.mdc-top-app-bar__title',
-  MENU_ICON_SELECTOR: '.mdc-top-app-bar__menu-icon',
-  ACTION_ICON_SELECTOR: '.mdc-top-app-bar__icon',
+  NAVIGATION_ICON_SELECTOR: '.mdc-top-app-bar__navigation-icon',
+  ACTION_ITEM_SELECTOR: '.mdc-top-app-bar__action-item',
 };
 
-export {strings};
+/** @enum {string} */
+const cssClasses = {
+  SHORT_CLASS: 'mdc-top-app-bar--short',
+  RIGHT_ACTION_ITEM_CLASS: 'mdc-top-app-bar--short__right-action-item',
```

Comment:
What's the meaning behind "right" action item, anyway? This term is probably a misnomer in RTL context so I'm wondering if there's something more semantic/agnostic we could use.